server/src/main/java/io/whitefox/api/deltasharing/server/restdto/TableResponseMetadata.java

```diff
@@ -0,0 +1,33 @@
+package io.whitefox.api.deltasharing.server.restdto;
+
+import io.whitefox.api.deltasharing.model.generated.MetadataResponse;
+import io.whitefox.api.deltasharing.model.generated.ProtocolResponse;
+
+public class TableResponseMetadata {
```

Comment:
same comments as: [DeltaTableMetadata.java](https://github.com/agile-lab-dev/whitefox/pull/63/files#diff-5f35af144830d4b9a403fa5a693e0963dd52688d2f30b15174a9e0afcdd5e9f6),
missin equals, hashcode, toString and private final